package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/snap-simple-keyring",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

Comment:
If we choose to release the site package as 2.0.0, should we bump this to a big value? This version doesn't really matter as it's not visible to users, it's really just used to tell the GitHub workflows that we have a new release available. But for other monorepos like `core` and `snaps` we have always bumped the major for the root package whenever there is a new release. And it doesn't have to be kept in sync with any other versions. So, in order to avoid confusion, perhaps this should be something like 50.0.0?

``` suggestion
  "version": "50.0.0",
```